lib/lambda_ethereum_consensus/state_transition/accessors.ex

```diff
@@ -3,6 +3,7 @@ defmodule LambdaEthereumConsensus.StateTransition.Accessors do
   Functions accessing the current beacon state
   """
 
+  alias ChainSpec
```

Comment:
does this line do something?

Reply:
Yeah, here `epochs_per_historical_vector = ChainSpec.get("EPOCHS_PER_HISTORICAL_VECTOR")`

Reply:
Elixir doesn't require module importing since modules are globally available. `alias` is just used to [locally rename modules](https://elixir-lang.org/getting-started/alias-require-and-import.html#alias), for example `alias SszTypes.Checkpoint` makes `%Checkpoint{}` possible, but without `alias`, you could still access it with its full name `%SszTypes.Checkpoint{}`.

Reply:
Gotcha. Thanks. 

Updated